.jenkins/continuous.groovy

```diff
@@ -280,7 +280,7 @@ pipeline {
                                 -DKokkos_ENABLE_EXAMPLES=ON \
                                 -DKokkos_ENABLE_TESTS=ON \
                                 -DKokkos_ENABLE_BENCHMARKS=ON \
-                                -DoneDPL_ROOT=/opt/intel/oneapi/2025.0 \
+                                -DoneDPL_ROOT=/opt/intel/oneapi/dpl/2022.7 \
```

Comment:
2022???

Reply:
They don't use the same version number, see https://github.com/uxlfoundation/oneDPL/releases/tag/oneDPL-2022.7.0-release. That's the version installed with `2025.0`.